content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -112,6 +113,17 @@ In the above example:
 The name of a StatefulSet object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
+
+### Minimum ready seconds
```

Comment:
I'd make the “Pod Selector” heading be third level (that text also needs a tidy BTW - Kubernetes < v1.8 is no longer supported) and put the  “Minimum ready seconds” heading just after the “Pod Selector“ section. That way, both of those are explaining components of the `.spec`.

Aside: it would be nice to document `volumeClaimTemplates` too.

Reply:
I've made the changes in a separate commit. Having said that, I think if the existing doc needs re-work, let's create a separate PR instead of using this PR.

Reply:
Let's not cover `volumeClaimTemplates` in _this_ PR.